package.json from npm appears to point to the wrong source directory

I was trying to get the demo to compile with TS 4.3, but I kept getting stuck with this error trying to `import * as tsickle from "tsickle";`

> Cannot find module 'tsickle' or its corresponding type declarations.

Seems like `typings` and `main` are set incorrect, or at least they don't match what actually gets installed by npm. Making the following changes seems to make everything happy.

```diff
{
  ... 
-  "main": "src/tsickle.js",
+  "main": "build/src/tsickle.js",
  ...
-  "typings": "src/tsickle.d.ts",
+  "typings": "build/src/tsickle.d.ts",
  "version": "0.40.0"
}
```

--- a/README.md
+++ b/README.md
@@ -208,5 +208,5 @@ From the master branch run:
 ```
 npm config set registry https://wombat-dressing-room.appspot.com
 npm login
-npm publish -- --registry https://wombat-dressing-room.appspot.com
+npm publish
 ```

--- a/demo/package.json
+++ b/demo/package.json
@@ -11,8 +11,8 @@
   },
   "dependencies": {
     "minimist": "^1.2.3",
-    "tsickle": "^0.37.0",
-    "typescript": "~3.5.3"
+    "tsickle": "^0.41.0",
+    "typescript": "~4.3"
   },
   "devDependencies": {
     "@types/minimist": "1.2.0",

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "build": "tsc ",
     "clean": "rm -r out",
     "lint": "tslint -c tslint.json -p tsconfig.json",
-    "test": "jasmine out/test/*.js && npm run lint"
+    "test": "jasmine out/test/*.js && (cd demo && yarn && tsc) && npm run lint"
   },
   "repository": {
     "type": "git",

